packages/components/pie-cookie-banner/test/visual/pie-cookie-banner.spec.ts

```diff
@@ -20,11 +20,6 @@ test.describe('PieCookieBanner - Visual tests`', () => {
 
         await pieCookieBannerComponent.clickManagePreferencesAction();
 
-        // This fixes modal being pushed down in screenshots
-        await page.$eval('dialog', (el) => {
-            el.style.top = '20px';
-        });
-
```

Comment:
We added this change as Percy has issues with rendering dialog elements nested in shadow DOM (pie-modal inside cookie banner) causing the modal’s backdrop and position to not render as expected. But, the workaround pushes the modal down on mobile screens, cutting off the button and removing it would result in more predictable screenshots imo